CHANGELOG.md

```diff
@@ -2,6 +2,8 @@ CHANGELOG
 =========
 
 ## HEAD (unreleased)
+- Add information about an in-flight operation to the stack command output, if applicable.
```

Comment:
As you can see from the other edits to the `CHANGELOG.md` file, it's useful to link to the bug that is being fixed so someone can drill into the specific item to learn more.